pkg/toolkit/app/http/config/dynamic/service.go

```diff
@@ -42,7 +42,7 @@ func (p *ReverseProxy) SetDefaults() {
 	}
 
 	if p.PassHostHeader == nil {
-		p.PassHostHeader = utils.Bool(true)
+		p.PassHostHeader = utils.Bool(false)
```

Comment:
why are you making PassHostHeader false by default? I don't remember if this change was required